docs/model_developers/model_design.rst

```diff
@@ -0,0 +1,80 @@
+Model Design Guidelines
+=======================
+
+This section provides design guidelines for experienced developers. 
+It is intended as a way of sharing experience and improving design. 
+
+.. warning::
+
+    We provide guidelines here. These are not absolute rules and not all rules are appropriate at all times.
+    Trust your own good judgement before anything else. 
+
+Overview
+--------
+
+South Bound Integration:
+
+#. Keep close to the API. Keep the structure of the inmanta model as close as possible to the API you model. Refrain from adding abstraction layers when doing pure integration.
+#. Prefer modeling relations as relations, avoid reference by string.
+#. Align indexes to the real world identity.
+#. Have index an aligned with resource identity
```

Comment:
What is the difference between these two lines?

Reply:
I haven't written those out, perhaps remove them for now?

Reply:
Yes. I would either remove it or make the difference more clear.

Reply:
done1